zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/distribution/CommandDistributionBehavior.java

```diff
@@ -232,8 +246,10 @@ private void startDistributing(
 
     sideEffectWriter.appendSideEffect(
         () -> {
-          interPartitionCommandSender.sendCommand(
-              partition, valueType, intent, distributionKey, commandValue);
+          if (!isPartitionScaling(partition)) {
```

Comment:
:question: When we were pairing we said that we could avoid calling the method `startDistributing` entirely if the partition was only in `desiredPartitions`. Is there a downside in doing that?